orchagent/vnetorch.cpp

```diff
@@ -547,11 +549,45 @@ bool VNetOrch::delOperation(const Request& request)
     return true;
 }
 
+bool VNetOrch::getVrfIdByVnetName(const std::string& vnet_name, sai_object_id_t &vrf_id)
```

Comment:
Should there be logic to handle vnet addition similar to onAddVR?

Reply:
Could yo please elaborate? I don't quite understand the comment here.